core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -1,45 +1,95 @@
 package zio.logging
 
+import zio._
 import zio.clock.{ currentDateTime, Clock }
 import zio.console.{ putStrLn, Console }
-import zio._
 
 object Logging {
-  type Logging = Has[Logging.Service]
-
-  trait Service {
-    def logger: Logger
-  }
+  type Logging = Has[Logger[String]]
```

Comment:
the import for this looks very awkward imo (import zio.logging.Logging.Logging) and is required everywhere

Reply:
yes I know... what do you recommend? package object with this type?

Reply:
I think that would be nicer, yeah :)

Reply:
Agreed, `Logging` should be in package object like ZIO core convention.

Reply:
done